Modules/Loadable/Transforms/Logic/vtkSlicerTransformLogic.cxx

```diff
@@ -307,7 +307,7 @@ void vtkSlicerTransformLogic::GetTransformedPointSamples(vtkPointSet* outputPoin
 //----------------------------------------------------------------------------
 void vtkSlicerTransformLogic::GetTransformedPointSamplesOnSlice(vtkPointSet* outputPointSet, vtkMRMLTransformNode* inputTransformNode,
   vtkMatrix4x4* sliceToRAS, double* fieldOfViewOrigin, double* fieldOfViewSize,
-  double pointSpacing, int pointGroupSize/*=1*/, int* numGridPoints/*=0*/, vtkPoints* samplePositions_RAS /*=NULL*/)
+  double pointSpacing, int pointGroupSize/*=1*/, int* numGridPoints/*=0*/, vtkPoints* samplePositions_RAS /*=nullptr*/)
```

Comment:
Looks likere there are  both  and `/* =nullptr */`, since these lines are updated, I suggest to consistently use `/*=nullptr*/`

Now should we use 

```
samplePositions_RAS/*=nullptr*/)
```

or 

```
samplePositions_RAS /*=nullptr*/)
``

or should we just get ride of these in a future commit ?

Cc: @lassoan

Reply:
I find these default parameter hints useful. Is the problem that spaces are not consistently added?

Reply:
> I find these default parameter hints useful

Agreed. I like them too, it makes code reading easier.

> Is the problem that spaces are not consistently added?

More of nitpick  than a problem per se, since we were updating the link we could have make spacing consistent. I like `samplePositions_RAS /*=nullptr*/)`

Reply:
I think they are helpful indeed. But maybe the style of these comments in arguments should be handled in other PR?
It would be many instances to fix that are not related to nullptr!